packages/image_picker/lib/image_picker.dart

```diff
@@ -50,12 +53,22 @@ class ImagePicker {
       throw ArgumentError.value(maxHeight, 'maxHeight cannot be negative');
     }
 
+    if (null == imageQuality) {
+      imageQuality = 100;
+    }
+
+    if (imageQuality != null && (imageQuality < 0 || imageQuality > 100)) {
```

Comment:
remove this "if" and use the assert I mentioned above.

Reply:
done